File: iosApp/iosApp/Models/UserProfile.swift
//
//  UserProfile.swift
//  BauchGlück
//
//  Created by Frederik Kohler on 15.07.24.
//  Copyright © 2024 orgName. All rights reserved.
//

import Foundation

struct UserProfile {
    let uid: String
    var firstName: String
    var lastName: String
    var email: String
    var surgeryDate: Date
    var mainMeals: Int = 3
    var betweenMeals: Int = 3
    var profileImageURL: String?
    var startWeight: Double
}

File: shared/src/commonMain/kotlin/model/UserProfile.kt
package model

data class UserProfile(
    val uid: String,
    val firstName: String,
    val lastName: String,
    val email: String,
    val surgeryDate: Long,
    val mainMeals: Int,
    val betweenMeals: Int,
    var profileImageURL: String?,
    val startWeight: Double,
    val waterIntake: Double,
    val waterDayIntake: Double
)